code/__HELPERS/unsorted.dm

```diff
@@ -2029,3 +2029,17 @@ var/mob/dview/dview_mob = new
 
 /proc/pass()
 	return
+	
+// \ref behaviour got changed in 512 so this is necesary to replicate old behaviour.
+// If it ever becomes necesary to get a more performant REF(), this lies here in wait
+// #define REF(thing) (thing && istype(thing, /datum) && (thing:datum_flags & DF_USE_TAG) && thing:tag ? "[thing:tag]" : "\ref[thing]")
+/proc/REF(input)

```

Comment:
be use `UID` instead of `REF`

Reply:
Moved the code for this under UID can you make sure what I did won't break anything? I tested it for a bit and it didn't seem to break anything major